tracing-extensions/modules/ballerina-sp-extension/src/main/assembly/dist.xml

```diff
@@ -0,0 +1,43 @@
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<assembly>
+    <id>distribution</id>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <formats>
+        <format>zip</format>
+    </formats>
+
+    <dependencySets>
+        <dependencySet>
+            <outputDirectory></outputDirectory>
+            <scope>runtime</scope>
+            <includes>
+                <include>org.wso2.sp:org.wso2.sp.open.tracer.client:jar</include>
+                <include>org.ballerinalang:ballerina-sp-extension:jar</include>
+                <include>org.wso2.carbon.analytics-common:org.wso2.carbon.databridge.agent</include>
+                <include>org.wso2.carbon.analytics-common:org.wso2.carbon.databridge.commons</include>
+                <include>org.apache.log4j.wso2:log4j</include>
+                <include>org.wso2.carbon.config:org.wso2.carbon.config</include>
+                <include>org.wso2.carbon.secvault:org.wso2.carbon.secvault</include>
+                <include>org.wso2.carbon.analytics-common:org.wso2.carbon.databridge.commons.thrift</include>
+                <include>io.opentracing:opentracing-api</include>
```

Comment:
I think this is already there in the distribution. Please check and include the jars that are not included in the default Ballerina distribution.